src/ResourceManager/Network/Commands.Network/Models/PSVirtualNetwork.cs

```diff
@@ -14,11 +14,12 @@
 
 namespace Microsoft.Azure.Commands.Network.Models
 {
-    using Newtonsoft.Json;
     using System.Collections.Generic;
     using System.Management.Automation;
-
-    public class PSVirtualNetwork : PSTopLevelResource
+    using Microsoft.Azure.Management.Internal.Network.Common;
+    using Newtonsoft.Json;
+    
+    public class PSVirtualNetwork : PSTopLevelResource, IResourceReference
```

Comment:
@muwaqar since we're making changes in the Network module, we will need to release a new version of their module along with Dns so users are able to access this updated type. Our mechanism for determining if a module has been updated or not is an entry in the change log, so you will need to update the Network change log with a short snippet about updating model types for compatibility with Dns cmdlets.